backend/backend/settings.py

```
@@ -2,90 +1,164 @@
-ALLOWED_HOSTS = ["*"]
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG =os.getenv("DEBUG", "False").lower() == "true"
+ENVIRONMENT = os.getenv("ENV")
+
+if ENVIRONMENT == 'PROD':
+    ALLOWED_HOSTS = [
+        
+    ]
+else:
+    ALLOWED_HOSTS = ["*"]  # Allow all hosts in development
```

Ruff (0.8.2)

Use ternary operator `ALLOWED_HOSTS = [] if ENVIRONMENT == "PROD" else ["*"]` instead of `if`-`else`-block

(SIM108)